doc/part-1-github-ui/github-contributions.md

```diff
@@ -8,6 +8,41 @@ description: 'PART 1: GITHUB UI'
 
 At this point, the focus is not about Git and Github best practices. The learning videos here will cover the basic things needed to get started with contributing to any project using the Github UI. What you will learn will also help you to tackle the tasks that have been provided for you to do after watching the learning videos.
 
+### You will learn the following in this section:
```

Comment:
#### Change request 2

This should not be an header. Just a statement that has a line of its own.

Reply:
Should I just keep this in bold instead of header?

Reply:
No, don't make it bold @aaryansr26